src/modules/klarna/klarna-api.ts

```diff
@@ -25,6 +31,13 @@ export type KlarnaMetadata = {
  * - North America:  https://api-na.playground.klarna.com/
  * - Oceania:  https://api-oc.playground.klarna.com/
  */
+const cleanupKlarnaUrl = (klarnaApiUrl: string) => {
+  const trimmedKlarnaApiUrl = klarnaApiUrl.trim();
+  const trimmedKlarnaApiUrlWithoutTrailingSlash = trimmedKlarnaApiUrl.endsWith("/")
+    ? trimmedKlarnaApiUrl.slice(0, -1)
+    : trimmedKlarnaApiUrl;
+  return trimmedKlarnaApiUrlWithoutTrailingSlash;
+};
```

Comment:
```suggestion
  return new URL(klarnaApiUrl).origin;
};
```
Suggestion: Seems to do the trick